tests/functionnal/CommonDBTM.php

```diff
@@ -221,4 +221,32 @@ public function testGetTable($classname, $tablename) {
          ->isEqualTo(\CommonDBTM::getTable($classname))
          ->isEqualTo($tablename);
    }
+
+   /**
+    * Provider for self::testGetTableField().
+    *
+    * @return array
+    */
+   protected function getTableFieldProvider() {
+
+      return [
+         [\Item_Devices::class, 'somefield', 'somefield'], // "static protected $notable = true;" case
```

Comment:
I'm not sure for the `$notable` case... For all other cases, we will get a unique field with the table name; but not with this one.
Shoudl we consider throwing an exception? I know exceptions are not properly handled but it may be better than just returning the field name.

Reply:
I agree. I just didn't do it as in is the how things are made on GLPI, but I think it is the best way to do.

Reply:
Done, and I reworked automatic tests to include a test on `*` keyword.